README.md

```diff
@@ -4,8 +4,10 @@ Track deprecation tasks
 
 ## How to use
 
-1. When an application, dataset, or both are requesting deprecation [create a new issue](https://github.com/agrc/cemetery/issues/new/choose)
-1. Choose the most applicable template from the buttons
-1. place an `[x]` in the relevant boxes and submit the issue
-1. tag anyone who might be a stakeholder in the deprecation
-1. follow through with the all of the items before closing the issue
+1. When an application, dataset, or both needs to be deleted or moved to a different long-term storage, [create a new issue](https://github.com/agrc/cemetery/issues/new/choose)
+1. Choose the `removing_data_or_service` template from the buttons
+1. Tag the three triagers (one from each team) in the triage section
```

Comment:
I think the dev team will handle this differently so we can leave that area blank and we will fill it in. do we want to edit the template to add the data team and casastre team triagers so the submitter doesn't have to know who they are?

Reply:
Yeah, that was what I was thinking, I just didn't/don't know who they'll be.